magda-gateway/views/admin.js

```diff
@@ -437,7 +437,10 @@ function imageConfig(body, name) {
 
 function request(method, url, body = null, contentType = undefined) {
     console.log(method, url, body);
-    const headers = {};
+    const headers = {
```

Comment:
Find this post:
https://stackoverflow.com/questions/49547/how-to-control-web-page-caching-across-all-browsers#answer-2068407
It seems the following headers can provide best compatibility coverage across different web browser:  
```
Cache-Control: no-cache, no-store, must-revalidate
Pragma: no-cache
Expires: 0
```

Reply:
👍